packages/lumx-core/style-dictionary/properties/core/typography.json

```diff
@@ -59,7 +59,7 @@
             "title1": {
                 "font-size": { "value": "40px" },
                 "font-weight": { "value": "var(--lumx-typography-font-weight-bold)" },
-                "line-height": { "value": "50px" }
+                "line-height": { "value": "1.5" }
```

Comment:
Line height is 1.5 by default -> just remove all typography line heights and it will inherit from 1.5 line height from the DS